cli/010-cli-datetime-format.md

```diff
@@ -0,0 +1,14 @@
+- Start Date: 2021-06-11
+- RFC PR:
+- Issue:
+
+### Format timestamp fields using a flag
+
+When printing data in `--table` or `--card` views, allow switching the format of the timestamp fields (ie. `google.protobuf.Timestamp`) 
+
+Add `--format-time` flag with possible values:
+- `relative` to print in a format `6 hours ago`. Use as default format in tty environment
+- `raw` to print in a format  `2021-06-16 00:21:24.472487365 +0000 UTC`.
+- `iso` to print in a format `2021-06-16T00:21:24Z` (ISO 8601, time.RFC3339). Use as default in non-tty environment
+- `tz=utc` to print for specific timezone
```

Comment:
or just `utc`/any other timezone and compare against timezones dictionary to tell to process for timezone?

Reply:
having `tz=` makes it easier to provide proper error message and print a list of possible values

Reply:
How does this work in combination with the other options?
`--format-time iso,tz=utc`?
I'm not sure `tz` fits here since it's not a format.

Reply:
i'll track TZ in a separate proposal and remove it from timestamp formatting